### DISCUSSION TITLE
Nuxt deployment

### DISCUSSION THREAD

**Discussion by JHerwig** [nuxt/nuxt#9578] [General]
Regarding to #9535 
On the [official nuxt page](https://nuxtjs.org/docs/2.x/deployment/deploying-to-21yunbox) are many deployment tools represented.  
Because the number is growing it would be nice to have a brief overview.

Some characteristics are:
- What can I deploy? (SSR, SSG, vanilla, ... )
- Where can I Host? (Own Server, Amazon, Google, GitHub, ... )
- Which requirements are there? (Licensing, any account, ... ) 

If you know one or more of these tools please add those properties.

**Comment by kissu** [comment]
We will indeed need to make a complete, exhaustive and updated presentation explaining all of the possible targets and possible builds. If my talk gets accepted, I'll showcase this during a conference.

If it doesn't, I'll guess we will make an explicit page in the docs with all the possible variants illustrated, like for [the plugins](https://nuxtjs.org/docs/2.x/directory-structure/plugins)!

As for the platforms, not sure if this is doable to maintain and explain all of them but usually, if:
- you do have an available server with a VPS (eg: DigitalOcean) it can do SSR
- if it is serverless (eg: [Cloudflare workers](https://workers.cloudflare.com/)), it'll be compatible with `edge-rendering` aka Nitro
- otherwise, it will be static and hosted on Netlify/Vercel/Surge/Github Pages

Some websites like [render.com](https://render.com/) are somewhat a mix between Heroku and a static Paas.
